python/plugins/processing/tests/testdata/qgis_algorithm_tests2.yaml

```diff
@@ -982,6 +982,20 @@ tests:
             bottom:
               precision: 7
 
+  - algorithm: native:linedensity
+    name: Line Density
+    params:
+      INPUT:
+        name: linedensity.gml|layername=linedensity
+        type: vector
+      PIXEL_SIZE: 1.0
+      RADIUS: 2.0
+      WEIGHT: weight
+    results:
+      OUTPUT:
+        hash: 37b153a98004a6656a27a8c5dcdb8dbefcd4af4938a512ec0cf5cd43
```

Comment:
These raster hashes are only good for checking whether the alg runs without errors -- they aren't any use for testing that no-one has broken the expected value.

I'd suggest adding specific checks to TestQgsProcessingAlgs instead, so that the actual output values are checked one by one.